snippets/lvms-disconnected-ImageSetConfig.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: SNIPPET
+.Example ImageSetConfiguration file for {lvms}
+[source,yaml,subs="attributes+"]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 4 <1>
+storageConfig: <2>
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata <3>
+    skipTLS: false
+mirror:
+  platform:
+    channels:
+    - name: stable-{product-version} <4>
+      type: ocp
+    graph: true <5>
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <6>
+    packages:
+    - name: lvms-operator <7>
+      channels:
+      - name: stable <8>
+  additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest <9>
+  helm: {}
+----
+<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
+<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
```

Comment:
> unless you are using the Technology Preview OCI feature.

What do I do if I am using the Technology Preview OCI feature?

Reply:
> When using the OCI feature, only the mirror.operators.catalog setting is available for use.
> The storageConfig setting is ignored in favor of the location passed in to the oc mirror command.

[This section](https://58494--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-oci-format_installing-mirroring-disconnected) has the above note about the `storageConfig` values. I didn't want to copy everything from the disconnected installation section, but it made sense to copy the callouts for the example yaml